model/flow/incorporated_result.go

```diff
@@ -35,7 +37,14 @@ func NewIncorporatedResult(incorporatedBlockID Identifier, result *ExecutionResu
 // ID implements flow.Entity.ID for IncorporatedResult to make it capable of
 // being stored directly in mempools and storage.
 func (ir *IncorporatedResult) ID() Identifier {
```

Comment:
The `IncorporatedResult` is used to store approval signatures when they are received: https://github.com/onflow/flow-go/blob/7c481533a9d41d4c435d2dbb9542e94452c07928/model/flow/incorporated_result.go#L26 When including `aggregatedSignatures` in the ID computation, this changes the ID of the entity, which should not be the case.

Reply:
💡 thanks for pointing this out @arrivets . I was not aware that RLP did skip private fields. 
Obviously, I didn't read the test you wrote: https://github.com/onflow/flow-go/blob/49fb4a12f45253a16441c43ae6da29068abd5d21/model/flow/incorporated_result_test.go#L12-L14 🤦 Apologies. 

Rolled back my changes to `incorporated_result.go`